libensemble/libE.py

```diff
@@ -270,8 +270,8 @@ def libE_mpi_worker(libE_comm, sim_specs, gen_specs, libE_specs):
     "Worker routine run at ranks > 0."
 
     from libensemble.comms.mpi import MainMPIComm
-    comm = MainMPIComm(libE_comm)
-    worker_main(comm, sim_specs, gen_specs, libE_specs, log_comm=True)
+    mpi_comm = MainMPIComm(libE_comm)
```

Comment:
This functions return is not mpi_comm. It is the libE comm object

Reply:
libE.py lines

    def __init__(self, comm):
        self.parent_comm = comm

could change the comm to mpi_comm:

    def __init__(self, mpi_comm):
        self.parent_comm = mpi_comm

Reply:
and maybe  comms_abort(comm):

Reply:
All of these were addressed in 7ecae7e8